katello/hooks/pre/15-check_java.rb

```diff
@@ -15,7 +15,7 @@ def error_exit(message, code)
   kafo.class.exit code
 end
 
-if module_enabled?('katello')
+if candlepin_enabled?
   java_version_string = `/usr/bin/java -version 2>&1`
   java_version = java_version_string.split("\n")[0].split('"')[1]
```

Comment:
On an unrelated note: I think this check itself should ensure JDK 1.8 is used since 1.7 is too old.